src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -495,3 +496,32 @@ The following steps are performed:
 | `--kubeconfig` `<string>`        | Absolute path(s) to the kubeconfig file(s)
 | `--namespace` `<string>`         | Namespace in which Submariner is installed (default "submariner-operator")
 | `--yes`                          | Automatically answer yes to confirmation prompt
+
+### Running `subctl diagnose` from a Pod in cluster
+
+As of release 0.12.0, a new `subctl` image containing the subctl binary is now available in submariner repos. This can be
+used to run subctl from a Pod within the cluster. `subctl diagnose` command output can be accessed from the Pod's logs.
+A new flag, `--in-cluster` has been added to support this use case.
+
+#### Example running `subctl diagnose` using a Job
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: submariner-diagnose
+  namespace: submariner-operator
+spec:
+  template:
+    spec:
+      containers:
+      - name: submariner-diagnose
+        image: quay.io/submariner/subctl:latest
+        command: ["subctl",  "diagnose", "all", "--in-cluster"]
+      restartPolicy: Never
+      serviceAccount: submariner-diagnose
+      serviceAccountName: submariner-diagnose
+  backoffLimit: 0
+```
+
+A similar template can also be used to create a CronJob that runs `subctl diagnose` periodically.
```

Comment:
```suggestion
A similar template can also be used to create a `CronJob` that runs `subctl diagnose` periodically.
```

I think this K8s object should be `CodeStyle`.

https://kubernetes.io/docs/contribute/style/style-guide/#code-style-inline-code

Although I see they don't don that in their own docs. :sweat_smile: 

https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/

So definitely a non-blocker, whatever you think I guess.